Gemfile

```diff
@@ -27,8 +27,9 @@ group :development do
   #gem 'metasploit-aggregator'
 end
 
-group :development, :test do
+# group :development, :test do
```

Comment:
This still needs to be restored.

Reply:
Thanks! Will pop this PR back into a draft until I've resolved all the comments 🎉